ui/views.py

```diff
@@ -69,7 +69,8 @@ class CollectionList(TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['collection_list'] = Collection.objects.all_admin(self.request.user)
-        context['form'] = forms.CollectionForm(initial={'owner': self.request.user.id})
+        context['form'] = (forms.CollectionForm(initial={'owner': self.request.user.id})
+                           if ui_permissions.is_staff_or_superuser(self.request.user) else None)
```

Comment:
We can just leave this key/value out of `context` if the permission doesn't exist, i.e.:

```python
if ui_permissions.is_staff_or_superuser(self.request.user):
    context['form'] = forms.CollectionForm(initial={'owner': self.request.user.id})
```

The template doesn't break if the key is missing

Reply:
will do, thanks for the review!